api_tests/tsconfig.json

```diff
@@ -1,15 +1,25 @@
 {
     "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
         "allowSyntheticDefaultImports": true,
-        "noImplicitAny": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "inlineSources": true,
         "lib": ["es5", "es2015", "es2017", "es6", "dom"],
-        "noEmit": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "noEmit": false,
+        "noImplicitAny": false,
+        "noUnusedLocals": false,
+        "noUnusedParameters": false,
+        "outDir": "dist",
         "resolveJsonModule": true,
-        "esModuleInterop": true,
-        "typeRoots": ["./node_modules/@types", "./types"]
-    }
+        "skipLibCheck": true,
+        "sourceMap": true,
+        "strict": false,
+        "target": "es6",
+        "declaration": true,
+        "typeRoots": ["./node_modules/@types", "./types"],
+        "types": ["node", "jest"]
+    },
+    "exclude": ["e2e", "dist", "lib", "dry", "tests"]
```

Comment:
Why are you excluding all the folders? Can jest now natively execute TS tests?

Reply:
`tsc` fails on all other directories :S